contrib/gabc.vim

```diff
@@ -1,7 +1,7 @@
 " Vim syntax file
 " Language:	Gabc gregorian chant notation
 " Maintainer:	Elie Roux <elie.roux@telecom-bretagne.eu>
-" Last Change:	2008 Nov 29
+" Last Change:	2016 Feb 11
```

Comment:
OK, I removed the "Maintainer" line.  I have tested this against most of the gabc files in the test repository.  I'm sure there are missing edge cases because of the looseness of the "syntax" of gabc, but I think it covers 90+% of cases.